bot/cogs/moderation/silence.py

```diff
@@ -27,7 +28,7 @@ def __init__(self) -> None:
     async def silence(
         self,
         ctx: commands.Context,
-        until: DurationConverter
+        until: Optional[DurationConverter]="15m"
```

Comment:
`DurationConverter` returns a `datetime.datetime` object, so the default parameter also must be a `datetime.datetime` object.

Reply:
Also, there should be a space after and before the `=`. The linter will fail, and therefore the build will also fail.

Reply:
I didn't knew about this, sorry 😕

Reply:
Don't worry. Just fix the code and I will merge the pull request. I suggest you to use `datetime.datetime.now()` and then add 15 minutes using `datetime.timedelta` inside the parameter list to make it as a default value. Something like (for example):
```py
def silence(until: datetime.datetime.now() + 15minutes):
    pass
```

Reply:
ok I will when I'm on pc